test/pstl_offload/memory/usm_memory_replacement.pass.cpp

```diff
@@ -37,6 +38,48 @@ void test_new(std::size_t count, NewArgs... new_args) {
     ::operator delete(ptr, new_args...);
 }
 
+void check_memory_ownership(const allocs &na, sycl::usm::alloc expected_type) {
+        EXPECT_TRUE(sycl::get_pointer_type(na.malloc_ptr, memory_context) == expected_type,
+            "Expect pointer from system allocation");
```

Comment:
It seems like the error message should be also changed to something like "Unexpected pointer type"

Reply:
Done.